packages/web/src/components/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+interface PopupModalProps {
+  openModalText: String;
+  header: String;
+  body: String;
+  openButtonVariant?: 'link' | 'outline' | (string & {}) | 'ghost' | 'solid' | 'unstyled';
```

Comment:
To keep it flexible we should use the type from the button props

```suggestion
  openButtonVariant?: ButtonProps['variant'];
```